backend/tests/test_all.py

```
@@ -0,0 +1,228 @@
+
+
+
+
+def test_root_endpoint_available():
+    response = client.get("/")
+    assert response.status_code in (200, 404)  
+def test_get_my_orders_endpoint_accessible():
+    response = client.get("/orders/me")
+    assert response.status_code in (401, 403)  
+
+def test_get_my_orders_details_endpoint_accessible():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_get_my_orders_details_endpoint_accessible`

Add return type annotation: `None`

(ANN201)